types/got/index.d.ts

```diff
@@ -69,7 +69,7 @@ declare class StdError extends Error {
 }
 
 declare const got: got.GotFn &
-    Record<'get' | 'post' | 'put' | 'patch' | 'head' | 'delete', got.GotFn> &
+    Record<'get' | 'post' | 'put' | 'patch' | 'head' | 'delete', 'extend', 'create', got.GotFn> &
```

Comment:
This maps `extend` and `create` to a `GotFn` which isn't what really what you want, or am I missing something?

Reply:
`got.extend` and `got.create` have different signatures than all the other functions, you can't simply add them to this Record.